sdk/endpoints/online/sample/online-inferencing-sample.ipynb

```diff
@@ -18,9 +19,9 @@
    "outputs": [],
    "source": [
     "#enter details of your AML workspace\n",
-    "subscription_id = '<SUBSCRIPTION_ID>'\n",
-    "resource_group = '<RESOURCE_GROUP>'\n",
-    "workspace = '<AML_WORKSPACE_NAME>'"
+    "subscription_id = 'd128f140-94e6-4175-87a7-954b9d27db16'\n",
+    "resource_group = 'ragargeastus2euap'\n",
+    "workspace = 'hanchi-test'"
    ]
```

Comment:
Can you please remove workspace and RG details from the notebook.

Reply:
Updated now. Missed one local commit.